roles/app/templates/application.yml.j2

```diff
@@ -45,3 +45,5 @@ STRIPE_ENDPOINT_SECRET: {{ stripe_endpoint_secret }}
 {% if bugsnag_key is defined %}
 BUGSNAG_API_KEY: {{ bugsnag_key }}
 {% endif %}
+
+OFN_FEATURE_CONNECT_AND_LEARN: "{{ feature_connect_and_learn | default('false') }}"
```

Comment:
What do you think of this way of setting defaults instead of `{% if key is defined %}`?

Reply:
The template is cleaner in this way, but the resulting YAML is full of env vars set to `false`... The problem that I see is that I prefer to check if an env var does exist instead of parsing the content for a literal `false`.

Reply:
We could have other env vars set like this, or additional feature toggles in the future, right? I think I'd prefer something more generic and reusable, like:

In Aus `config.yml`:
```
instance_env_vars:
   - "OFN_FEATURE_CONNECT_AND_LEARN: true"
```

In `application.yml.j2`:
```
{% for instance_env_var in instance_env_vars %}
{{ instance_env_var }}
{% endif %}
```

Reply:
Uhh, interesting. I like that. So in many cases we could actually skip the Ansible variable. For example we don't define `feature_connect_and_learn` and just define the env var in the config.

We are basically storing arbitrary lines in an array and then printing them in a loop. We could also just define a text block in the config that is included.

In Aus `config.yml`:
```yml
application_env_vars: |
  OFN_FEATURE_CONNECT_AND_LEARN: true
```

In `application.yml.j2`:
```yml
{{ application_env_vars }}
```

Of course, this method can only be used for variables that are not used in other parts of the Ansible code. The downside is that we have two ways of setting variables. But I think it's okay.

Should we do that in a separate pull request?

Reply:
There's lots of ways to set lists or blocks in variables, I think it's ok to mix and match. 

We may as well do it in this PR, I think.

Reply:
Okay, please see my new commit, @Matt-Yorkley.